staging/src/k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap/default.go

```diff
@@ -305,40 +305,13 @@ var (
 		},
 	)
 	SuggestedFlowSchemaSystemLeaderElection = newFlowSchema(
-		"system-leader-election", "leader-election", 100,
+		"leader-election", "leader-election", 100,
```

Comment:
note: the name change will not leave a dangled flowschema during cluster upgrade, the auto updater will treat the old object as unwanted and will remove it (given that the annotation is present which it always should)

Reply:
Yes - I'm aware of it.
But to double check - will the old one be deleted first or the new one created first?

If the former, we need to fix this problem, if the latter this is safe, right?

Reply:
removing is the final step, so i expect the new one created first and then the old one deleted

Reply:
how do the old API server and the new API server reconcilers interact when they disagree on the names or content of default items?

Reply:
It looks like the `n-1` API server will work to create the old flowschema and remove this new flowschema, and the `n` API server will work to create this new flowschema and remove the old flowschema

Has that interaction been tested? What is the effect of the flowschemas fluttering back and forth like that?

Reply:
> Has that interaction been tested? What is the effect of the flowschemas fluttering back and forth like that?

I tested it locally and it worked, but thinking more about this actually is racy and I might have been lucky.
I was thinking more in the context of ensuring that this reconciliation won't overload and we have this backoff mechanism (which fired correctly). But there is a race possible when we will actually all rules.

So let me revert its name to the previous one, which then removes this problem.